### DISCUSSION TITLE
Scaffolded item not functioning properly

### DISCUSSION THREAD

**Discussion by jasperRigters** [phalcon/cphalcon#15500] [Q&A] [answered]
Hi!

I am completely new to Phalcon, and after installing it along with the dev-tools I scaffolded a database table users.

It created a model and a controller, along with some views for the CRUD.

Now if I try to add a user using the provided interface the url will jump to http://0.0.0.0:8000/users/users/create (note the double users) and the user will not be added to the database. 

If I try to search for a user, it will work the first try. With the url being http://0.0.0.0:8000/users/search?id=1&name=&role_id= (I searched for a id of 1).
But if I click the search button a second time the requested ulr will be http://0.0.0.0:8000/users/users/search?id=&name=&role_id= (note the double users again).
I am not able to figure out what is causing this, I have done nothing but created a SQL table and rand the scaffold command in the dev-tools. 

Please help

**Comment by jasperRigters** [comment, marked as answer]
Found the solution, the culprit resides in the scaffolded form:
`<form action="users/create" class="form-horizontal" method="post">`

Should be:
`<form action="/users/create" class="form-horizontal" method="post">`

Should I report this as a bug or something?

**Reply by wurst-hans** [reply]
This is not a bug. Using a relative URL like in your first example `users/create` this means relative to current URL. When the URL in your browser is `<host>/users/create` already, without a trailing slash, your browser thinks that the current **folder** is `/users` and will resolve the form action to `/users/users/create`. Because of that you should  use absolute paths (i.e. with a leading slash) as URL, always.

**Reply by jasperRigters** [reply]
Shouldn't the dev tools have the leading slash then?
